lib/workload/stateful/database/component.ts

```diff
@@ -158,6 +163,20 @@ export class Database extends Construct {
       }),
     });
 
+    new sm.SecretRotation(this, 'MasterDbSecretRotation', {
+      application: sm.SecretRotationApplication.POSTGRES_ROTATION_SINGLE_USER,
+      // From default: https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds.DatabaseSecret.html#excludecharacters
+      excludeCharacters: '" %+~`#$&()|[]{}:;' + `<>?!'/@"\\")*`,
+      secret: dbSecret,
+      target: this.cluster,
+      automaticallyAfter: Duration.days(1),
```

Comment:
Should this not use the `secretRotationSchedule` parameter of the config?

Reply:
Ah, miss reverting this after testing! Thanks